### ISSUE TITLE
Enhancement: Add more mynocks to the mining area

### ISSUE THREAD

**Issue by Zed287** [zunath/SWLOR_NWN#243]
As titled

The questline to kill mynocks will be pretty difficult for new players, especially launch if there are only a total of 5 mynocks in the basement area.

I would suggest adding some to the ore areas and around the facility in general.

**Comment by taelon875** [comment]
There are more then five, i think there are 10 last count I did, and only 5 are needed for the quest. (if I recall we lowered it)

**Comment by Zed287** [comment]
Right, but It may be important to add more, We have to expect atleast 10 or so players coming down to the mines
